openshift/templates/lup-api-mongodb-dev-deploy-old.yaml

```diff
@@ -0,0 +1,283 @@
+              'f:volumes':
+                .: {}
+                'k:{"name":"landuseplanning-api-mongodb-dev"}':
+                  .: {}
+                  'f:name': {}
+                  'f:persistentVolumeClaim':
+                    .: {}
+                    'f:claimName': {}
+    - manager: openshift-controller-manager
+      operation: Update
+      apiVersion: apps.openshift.io/v1
+      time: '2022-03-30T19:36:57Z'
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          'f:template':
+            'f:spec':
+              'f:containers':
+                'k:{"name":"landuseplanning-api-mongodb-dev"}':
+                  'f:image': {}
+          'f:triggers': {}
+        'f:status':
+          'f:conditions':
+            .: {}
+            'k:{"type":"Available"}':
+              .: {}
+              'f:type': {}
+            'k:{"type":"Progressing"}':
+              .: {}
+              'f:type': {}
+          'f:details':
+            .: {}
+            'f:causes': {}
+            'f:message': {}
+          'f:latestVersion': {}
+    - manager: openshift-controller-manager
+      operation: Update
+      apiVersion: apps.openshift.io/v1
+      time: '2025-02-18T21:28:05Z'
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:status':
+          'f:availableReplicas': {}
+          'f:conditions':
+            'k:{"type":"Available"}':
+              'f:lastTransitionTime': {}
+              'f:lastUpdateTime': {}
+              'f:message': {}
+              'f:status': {}
+            'k:{"type":"Progressing"}':
+              'f:lastTransitionTime': {}
+              'f:lastUpdateTime': {}
+              'f:message': {}
+              'f:reason': {}
+              'f:status': {}
+          'f:observedGeneration': {}
+          'f:replicas': {}
+          'f:unavailableReplicas': {}
+          'f:updatedReplicas': {}
+      subresource: status
+  namespace: e8b9ad-dev
+  labels:
+    app: landuseplanning-api-mongodb-dev
+    app.kubernetes.io/component: landuseplanning-api-mongodb-dev
+    app.kubernetes.io/instance: landuseplanning-api-mongodb-dev
+    app.kubernetes.io/part-of: landuseplanning-api-dev
+    app.openshift.io/runtime-namespace: openshift
+    app.openshift.io/runtime-version: latest
+spec:
+  strategy:
+    type: Rolling
+    rollingParams:
+      updatePeriodSeconds: 1
+      intervalSeconds: 1
+      timeoutSeconds: 600
+      maxUnavailable: 25%
+      maxSurge: 25%
+    resources: {}
+    activeDeadlineSeconds: 21600
+  triggers:
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+          - landuseplanning-api-mongodb-dev
+        from:
+          kind: ImageStreamTag
+          namespace: openshift
+          name: 'mongodb:latest'
+        lastTriggeredImage: 'image-registry.openshift-image-registry.svc:5000/openshift/mongodb@sha256:8dc9932eabd30f2ee4d0b882500c65b48f3c2fbbd85bfd51d0d18410af404a39'
+    - type: ConfigChange
+  replicas: 0
```

Comment:
Should this be 0? If we need to quickly restore this, the 0 wouldn't spin up any pods by default right?

Reply:
Ah I think this happened because I copied the data from the YAML after we spun down the pods to 0.